Bottom: c44ebc059c96c1c05f6eeeb6752d9e1f4ebfa81b
Top:    12a4689d5e48fdb371b9308c7bca0631d693ec98
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-11 15:24:57 +0100

Refresh of avoid-computing-internal-point

---

diff --git a/src/polygon.c b/src/polygon.c
index af92d10..0d484c0 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -111,7 +111,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 10
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
